src/connect.lua

```diff
@@ -164,20 +172,20 @@ local function connect<StoreState, Props, MappedStatePartialProps, MappedDispatc
 			end
 
 			local mappedStoreDispatch: any
-			if mapDispatchType == "table" then
+			if isCallable(mapDispatchToProps) then
```

Comment:
I swapped the order of the if / elseif checks here because a table can still be callable